ChatService/src/controllers/message.controller.js

```diff
@@ -50,18 +79,49 @@ class MessageController {
   deleteMessage = async (req, res) => {
     try {
       const { messageId } = req.params;
-      const { type } = req.query; 
+      console.log(req.user.id);
+      const userId  = req.user.id;
+      console.log('User ID:', userId);
+      const { deleteType } = req.body;
+      
+      if (!deleteType || !['everyone', 'justme'].includes(deleteType)) {
```

Comment:
```suggestion
      const invalidType = !deleteType || !['everyone', 'justme'].includes(deleteType)
      if (invalidType) {
```
Hạn chế để logic trong condition if. Để dễ debug hơn